client/src/websocket/socket-events.js

```diff
@@ -0,0 +1,20 @@
+const removeListeners = (socket) => {
+    socket.off('connect');
+    socket.off('disconnect');
+    socket.off('connect_error');
+}
```

Comment:
![Screen Shot 2021-02-01 at 5 08 21 PM](https://user-images.githubusercontent.com/45152761/106524745-00c79180-64b1-11eb-92a1-fbd853d0839a.png)

I included that part because I was seeing an increasing number of console logs from the socket.on connect and socket.on disconnect listeners when I repeatedly logged in and out. Manually removing the listeners seemed to fix it.